### ISSUE TITLE
Implement error logging and reporting

### ISSUE THREAD

**Issue by ruler225** [QueueTime/qtime-backend#46]
Errors in the backend are currently unreported and unlogged. A framework such as [Sentry](https://docs.sentry.io/platforms/python/) should be used to log and automatically report server errors and send them via email (or some other method)

**Comment by benjaminkostiuk** [comment]
This is such a good idea. I don't know if we'll actually have time or resources to implement it but if we keep this around after the expo for sure this is a great idea.

**Comment by ruler225** [comment]
Yeah agreed this is definitely nice to have but lower priority at this stage.
